urbit/umbrel-app.yml

```diff
@@ -35,24 +35,19 @@ torOnly: false
 submitter: ~mopfel-winrux
 submission: https://github.com/getumbrel/umbrel/pull/1246
 releaseNotes: >-
-  -   Revamped Urbit’s terminal stack. This release contains the first
-      phase of a longer-running project towards a next-generation terminal
-      experience. Highlights in this release include:
-      
-      -   Support for 2D cursor movement, and by extension 2D screen
-          drawing.
-          
-      -   Minimal mouse-click support.
-      
-      -   Faster pasting of large texts.
-      
-      -   Informal printfs (~&) now get rendered in grey, to visually
-          distinguish them from formal dill output.
-          
-  -   Updated versioning scheme for pre-release binaries.
-  
-  
-  -   The code for the Vere runtime has been moved out of the urbit/urbit
-      repository, and now lives in urbit/vere.
-
-  Full release notes here: https://github.com/urbit/vere/releases/tag/vere-v1.17
+
+  Adds runtime support for the doccords system, which is documentation

```

Comment:
Looks much better, thank you!

![Screenshot 2023-02-02 at 02 15 35](https://user-images.githubusercontent.com/1112574/216214595-df4449a7-6175-437c-a177-bd79110910b5.png)

Can you remove the empty line above this? After that, we're good to go.

Reply:
Gotcha, I misunderstood what you meant in the first comment. Removed